package.json

```diff
@@ -1,63 +1,53 @@
 {
   "name": "@percy/migrate",
   "description": "Easily upgrade and migrate your Percy SDK to the latest version",
-  "version": "0.11.0",
+  "version": "1.0.0",
   "license": "MIT",
   "main": "dist/index.js",
+  "type": "module",
   "bin": {
-    "percy-migrate": "bin/run"
+    "percy-migrate": "bin/run.cjs"
   },
   "files": [
     "bin",
     "dist",
-    "transforms",
-    "oclif.manifest.json"
+    "transforms"
   ],
   "engines": {
-    "node": ">=12"
+    "node": ">=14"
+  },
+  "@percy/cli": {
+    "commands": [
+      "./dist/index.js"
+    ]
```

Comment:
This actually isn't technically needed. Basically, this is only necessary for the `@percy/cli` package to pick this up as a plugin under the `percy migrate` command. Since this is standalone package with it's own command, it's not necessary, but we could still leave it if we want to allow that plugin behavior. You can even see this command appear in the list of `percy` commands in a screenshot in my other comment.

Reply:
Actually kinda raises an interesting question: should we? I don't really mind either way but this package was always kinda set out to be stand alone 🤨

Reply:
Yeah. I feel like this is the type of package that you'd likely never want to actually install. Just a one time use unless it's a monorepo or multiple projects, but in that case npx should cache it. 

So let's remove this and demote it from being a plugin